start:
  pose:
    position: {x: 2.71006066106, y: 2.12491014098, z: 0.0}
    orientation: {x: 0.0, y: 0.0, z: 0.752734065, w: 0.658324712318}

finish:
  pose:
    position: {x: 2.71006066106, y: 2.12491014098, z: 0.0}
    orientation: {x: 0.0, y: 0.0, z: 0.752734065, w: 0.658324712318}


wait:
  pose:
    position: {x: 2.71006066106, y: 2.12491014098, z: 0.0}
    orientation: {x: 0.0, y: 0.0, z: 0.752734065, w: 0.658324712318}

wait_centre:
  pose:
    position: {x: 2.71006066106, y: 2.12491014098, z: 0.0}
    orientation: {x: 0.0, y: 0.0, z: 0.752734065, w: 0.658324712318}


lift_centre:
  pose:
    position: {x: 2.71006066106, y: 2.12491014098, z: 0.0}
    orientation: {x: 0.0, y: 0.0, z: 0.752734065, w: 0.658324712318}


lift:
  pose:
    position:
      x: 3.38024333356
      y: -0.707694082116
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.59235272545
      w: 0.805678750279

floor:
  status: "unknown"
  number: 0

counter_door:
  counter: 0

in_lift:
  status: False


counter_lift:
  counter: 0

